README.md

```diff
@@ -1,52 +1,136 @@
 # React A11yDialog
 
-react-a11y-dialog is a React component for [a11y-dialog](https://github.com/edenspiekermann/a11y-dialog) meant to ease the use of accessible dialog windows in React applications.
+react-a11y-dialog is a React component for [a11y-dialog](https://github.com/edenspiekermann/a11y-dialog) relying on [React portals](https://reactjs.org/docs/portals.html) to ease the use of accessible dialog windows in React applications.
+
+*Note: for React versions **before** 16, use `react-a11y-dialog@2.0.0`.*
 
 ## Install
 
 ```
 npm install --save react-a11y-dialog
 ```
 
-## Example
+## API
+
+* **Property name**: `id`
+* **Type**: string
+* **Mandatory**: true
+* **Default value**: —
+* **Description**: The HTML `id` attribute of the dialog element, internally used by a11y-dialog to manipulate the dialog.
```

Comment:
“Manipulate”? What's going on here exactly? “to reference the dialog element”?

Reply:
To control.